examples/ilastik_segmentation.py

```diff
@@ -0,0 +1,145 @@
+"""
+.. _tutorial_ilastik_segmentation:
+
+Using ilastik in starfish
+=========================
+
+In order to create a cell by gene expression matrix from image-based transcriptomics data, RNA
+spots must be assigned to cells. Segmenting a microscopy image into single cells is either
+time-consuming when done manually or inaccurate when done with thresholding and watershed. The
+advent of deep learning models for image segmentation promises a solution that is both fast and
+accurate.
+
+Starfish currently has built-in functionality to support `ilastik <https://www.ilastik.org/>`_,
+a segmentation toolkit that leverages machine-learning. Ilastik has a Pixel Classification
+workflow that performs semantic segmentation of the image, returning probability maps for each
+label (e.g. cells). To use ilastik in starfish first install ilastik locally and follow the
+Pixel Classification workflow using the GUI to train a classifier, which is saved in an ilastik
+project file.
+
+There are two options for using ilastik in starfish. The first is running the trained pixel
+classifier on images outside of starfish to generate a probability map that can then be loaded in
+starfish as an :py:class:`.ImageStack`. The second is to run the pixel classifier from within
+starfish by calling out to ilastik and passing it the trained classifier. The results will be the
```

Comment:
```suggestion
starfish by calling out to ilastik and passing it to the trained classifier. The results will be the
```

Reply:
My understanding is that the trained classifier is passed to ilastik (the "it" in my sentence). Tony's comment makes "it" the image stack, which is passed to the trained classifier. Which is correct?

`subprocess.check_call(
                [self.ilastik_executable,
                 '--headless',
                 '--project',
                 self.ilastik_project,
                 "--output_filename_format",
                 output_file,
                 dapi_file], env={})
`

Reply:
Sorry, you're right.  Original sentence is better.